src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+
+/* okay */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
```

Comment:
The `.stars` class is well named, indicating it's a component dealing with stars, which is good for readability and maintainability.